Update windows-10-multisession-faq.md (#72374)
Azure/azure-content: articles/virtual-desktop/windows-10-multisession-faq.md

Windows 10 Enterprise multi-session can't run in on-premises production environments because it's optimized for the Windows Virtual Desktop service for Azure. It's against the licensing agreement to run Windows 10 Enterprise multi-session outside of Azure for production purposes. Windows 10 Enterprise multi-session won't activate against on-premises Key Management Services (KMS).

> ## Is it possible to upgrade a Windows 10 VM to Windows 10 Enterprise multi-session?

Reviewer: ```suggestion
## Can I upgrade a Windows 10 VM to Windows 10 Enterprise multi-session?
```

> No. You cannot upgrade an existing Windows 10 VM (for example Windows 10 Professional or Enterprise) to Windows 10 Enterprise multi-session. In addition if you deploy a Windows 10 Enterprise multi-session VM then update product key to another edition there is not a way to switch the VM back to Windows 10 Enterprise multi-session, you will have to re-deploy the VM.

Reviewer: ```suggestion
No. It's not currently possible to upgrade an existing virtual machine (VM) that's running Windows 10 Professional or Enterprise to Windows 10 Enterprise multi-session. Also, if you deploy a Windows 10 Enterprise multi-session VM and then update product key to another edition, you won't be able to switch the VM back to Windows 10 Enterprise multi-session and will need to redeploy the VM.
```

> No. It's not currently possible to upgrade an existing virtual machine (VM) that's running Windows 10 Professional or Enterprise to Windows 10 Enterprise multi-session. Also, if you deploy a Windows 10 Enterprise multi-session VM and then update product key to another edition, you won't be able to switch the VM back to Windows 10 Enterprise multi-session and will need to redeploy the VM.

Reviewer: ```suggestion
No. It's not currently possible to upgrade an existing virtual machine (VM) that's running Windows 10 Professional or Enterprise to Windows 10 Enterprise multi-session. Also, if you deploy a Windows 10 Enterprise multi-session VM and then update the product key to another edition, you won't be able to switch the VM back to Windows 10 Enterprise multi-session and will need to redeploy the VM.
```
